docs/external-node.md

```diff
@@ -561,3 +562,39 @@ interfaces will be added in the future.
 
 `ExternalNode` name must be unique in the `cluster` scope even though it is
 itself a Namespaced resource.
+
+## Known issues
+
+After installing `antrea-agent` on VM, `antrea-agent` fails to attach the VM's
+network interface to the OVS bridge if the VM is rebooted. On Windows VM,
+network connectivity might be lost after reboot.
+
+As a workaround to [issue #4122](https://github.com/antrea-io/antrea/issues/4122),
+manually remove OVS configurations and then restart `antrea-agent` after the VM
+is rebooted.
+
+To remove OVS configurations and restart `antrea-agent` on Linux VM,
+
+```shell
+sudo systemctl stop antrea-agent
+sudo ovs-vsctl del-br br-int
+sudo systemctl start antrea-agent
+```
+
+To remove OVS configurations and restart `antrea-agent` on Windows VM,
+
+```powershell
+$adapterName="Ethernet 0"
+Stop-Service antrea-agent
+ovs-vsctl.exe del-br br-int
+Remove-VMSwitch -ComputerName $(hostname.exe) antrea-switch -Force
+Rename-NetAdapter -Name "$adapterName~" -NewName "$adapterName"
+Start-Service antrea-agent
+```
+
+Note:
+
+- Variable `$adapterName` is configured with the `ExternalNode` interface name.
+- For the workaround, a secondary network interface may be needed to RDP into
```

Comment:
"You may need a separate network interface to RDP into the Windows VM to run these commands, since.."

Reply:
updated